src/main/resources/application.yml

```diff
@@ -15,8 +15,8 @@ org:
       logging:
         LoggingSystem: none
 jar:
-  path:
```

Comment:
It is only whites pace changes in this file. It should not be part of the pull request.